_stbt/core.py

```diff
@@ -1326,7 +1326,12 @@ def is_screen_black(self, frame=None, mask=None, threshold=None,
             else:
                 imglog.imwrite('non-black-regions-after-masking', greyframe)
 
-        return IsScreenBlackResult(frame, maxVal == 0)
+        result = IsScreenBlackResult(frame, maxVal == 0)
+        if result:
+            debug("is_screen_black: Found black screen: %s" % result)
+        else:
+            debug("is_screen_black: Didn't find black screen: %s" % result)
```

Comment:
Log the threshold, region and frame timestamp and mask filename here too.

Reply:
Done. I've also logged the actual maximum value (intensity) seen, which is useful if you need to adjust your threshold.